api/alert_channels_jira_cloud_server.go

```diff
@@ -24,6 +24,34 @@ import (
 	"strings"
 )
 
+type jiraIssueGrouping int
+
+const (
+	NoJiraIssueGrouping jiraIssueGrouping = iota
+	EventsJiraIssueGrouping
+	ResourcesJiraIssueGrouping
+)
+
+var JiraIssueGroupings = map[jiraIssueGrouping]string{
+	NoJiraIssueGrouping:        "",
```

Comment:
nit: we have used `None` in other "enums", should we follow standard?
```suggestion
	NoneJiraIssueGrouping:        "",
```

Reply:
how about `NunyaJiraIssueGrouping`

Reply:
Fixed.